subworkflows/local/prepare_genome.nf

```diff
@@ -2,37 +2,41 @@
-include { UNTAR as UNTAR_HISAT2_INDEX } from '../../modules/nf-core/modules/untar/main'                     addParams( options: params.hisat2_index_options )
-include { UNTAR as UNTAR_SALMON_INDEX } from '../../modules/nf-core/modules/untar/main'                     addParams( options: params.index_options        )
-include { GFFREAD                     } from '../../modules/nf-core/modules/gffread/main'                   addParams( options: params.gffread_options      )
-include { STAR_GENOMEGENERATE         } from '../../modules/nf-core/modules/star/genomegenerate/main'       addParams( options: params.star_index_options   )
-include { HISAT2_EXTRACTSPLICESITES   } from '../../modules/nf-core/modules/hisat2/extractsplicesites/main' addParams( options: params.hisat2_index_options )
-include { HISAT2_BUILD                } from '../../modules/nf-core/modules/hisat2/build/main'              addParams( options: params.hisat2_index_options )
-include { SALMON_INDEX                } from '../../modules/nf-core/modules/salmon/index/main'              addParams( options: params.salmon_index_options )
+    GUNZIP as GUNZIP_ADDITIONAL_FASTA  } from '../../modules/nf-core/modules/gunzip/main'                    addParams( options: params.genome_options        )
+include { UNTAR as UNTAR_BBSPLIT_INDEX } from '../../modules/nf-core/modules/untar/main'                     addParams( options: params.bbsplit_untar_options )
+include { UNTAR as UNTAR_STAR_INDEX    } from '../../modules/nf-core/modules/untar/main'                     addParams( options: params.star_index_options    )
+include { UNTAR as UNTAR_RSEM_INDEX    } from '../../modules/nf-core/modules/untar/main'                     addParams( options: params.index_options         )
+include { UNTAR as UNTAR_HISAT2_INDEX  } from '../../modules/nf-core/modules/untar/main'                     addParams( options: params.hisat2_index_options  )
+include { UNTAR as UNTAR_SALMON_INDEX  } from '../../modules/nf-core/modules/untar/main'                     addParams( options: params.index_options         )
+include { GFFREAD                      } from '../../modules/nf-core/modules/gffread/main'                   addParams( options: params.gffread_options       )
+include { STAR_GENOMEGENERATE          } from '../../modules/nf-core/modules/star/genomegenerate/main'       addParams( options: params.star_index_options    )
+include { HISAT2_EXTRACTSPLICESITES    } from '../../modules/nf-core/modules/hisat2/extractsplicesites/main' addParams( options: params.hisat2_index_options  )
+include { HISAT2_BUILD                 } from '../../modules/nf-core/modules/hisat2/build/main'              addParams( options: params.hisat2_index_options  )
+include { SALMON_INDEX                 } from '../../modules/nf-core/modules/salmon/index/main'              addParams( options: params.salmon_index_options  )
```

Comment:
I'll just use this excellent opportunity to highlight why I believe aligning is a bad idea: if this wasn't aligned, the diff would be a single line :stuck_out_tongue: :running_man:

Reply:
I know....and I knew you would say that...I can't be arsed to left align everything. But the important thing is I won't sleep if we don't pick either - middle ground is a no no 😆